api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -35,9 +40,31 @@
 )
 CensusDirectory = Dict[CensusVersionName, Union[CensusVersionName, CensusVersionDescription]]
 
+CensusMirrorName = str  # name of the mirror
+CensusMirror = TypedDict(
+    "CensusMirror",
+    {
+        "provider": str,  # provider of the mirror. Only S3 is supported in this version.
+        "base_uri": str,  # name of the bucket or resource
+        "region": Optional[str],  # region of the bucket or resource
+    },
+)
+
+CensusMirrors = Dict[CensusMirrorName, Union[CensusMirrorName, CensusMirror]]
+
+ResolvedCensusLocator = TypedDict(
```

Comment:
Suggest renaming this `ResourceLocator` or `ResolvedResourceLocator`, as it is used for both Census and H5AD resources, not just the Census.